hq/public/stylesheets/main.css

```diff
@@ -142,13 +207,13 @@ nav {
   margin: 0 auto;
 }
 
-@media (min-width: 740px) {
+@media (min-width: 940px) {
   .sg-container {
     grid-template-columns: repeat(2, 1fr);
   }
 }
 
-@media (min-width: 1140px) {
+@media (min-width: 1400px) {
```

Comment:
Was running the branch with the VPC and cloudformation stack details and noticed that the SG cards now need a bit more horizontal room, so have given them some breathing space as part of this PR.